addons/account/models/account_journal_dashboard.py

```diff
@@ -560,7 +560,8 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
                 domain=[
                     *self.env['account.move']._check_company_domain(self.env.companies),
                     ('journal_id', 'in', sale_purchase_journals.ids),
-                    ('to_check', '=', True),
+                    ('checked', '=', False),
+                    ('state', '=', 'posted'),
```

Comment:
How is it that we now have more criterias ? Also applies elsewhere, I'm thinking of the draft state for instance. This is probably legit, but the commit message could mention it.

Reply:
Since we are inverting the logic of the field, and that it's checked on post, we have all the draft entries that are displayed in the to_check filter. So we decided to put a constraint on the state so that we only show posted entries :smile: Will add that in the commit message :smile: